tests/js/spec/views/organizationGroupDetails/groupTags.spec.jsx

```diff
@@ -20,6 +20,7 @@ describe('GroupTags', function() {
       <GroupTags
         group={group}
         query={{}}
+        environments={['dev']}
```

Comment:
So we should still test that the API request gets called with the environment url params --
`MockApiClient.addMockResponse()` returns a jest mock function, which you can assert gets called with the right args.